### PR TITLE
Adding cmip6_preprocessing

### PR DESCRIPTION
**Pull request by jbusecke**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there (I am the sole maintainer)

### CODE REVIEW

File: recipes/cmip6_preprocessing/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - pytest-runner
```
Comment (scopatz) [inline recipes/cmip6_preprocessing/meta.yaml]: Is this needed for install?

File: recipes/cmip6_preprocessing/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d34fa682b53fd353ef83c7cddddb174e26e1ecfc63cbccbac99e6157f2370750
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (scopatz) [inline recipes/cmip6_preprocessing/meta.yaml]: Please remove empty build block

File: recipes/cmip6_preprocessing/meta.yaml (hunk 1 above)
Comment (jbusecke) [reply recipes/cmip6_preprocessing/meta.yaml]: I believe so. If I remove either, the local tests fail.

File: recipes/cmip6_preprocessing/meta.yaml (hunk 2 above)
Comment (jbusecke) [reply recipes/cmip6_preprocessing/meta.yaml]: Done

**Comment by jbusecke** [comment]
Many thanks for the quick review @scopatz. I made the requested changes.

File: recipes/cmip6_preprocessing/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  summary: 'Some code to homogenize naming and parse variables in a xgcm compatible xarray dataset'
+
+  description: |
+    Frustrated with how 'dirty' CMIP6 data still is? Do you just want to run a
+    simple (or complicated) analysis on various models and end up having to
+    write logic for each seperate case? Then this package is for you.
+
+    Developed during the cmip6-hackathon this package provides utility
+    functions that play nicely with intake-esm.
+  # doc_url: https://simplejson.readthedocs.io/
```
Comment (scopatz) [inline recipes/cmip6_preprocessing/meta.yaml]: please remove comments

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**